src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -1704,4 +1704,43 @@ public String getTextFromComment(Text commentText) {
         }
         return SanitizationHelper.sanitizeForCsv(comment.toString());
     }
+
+    /**
+     * Checks if bundle contains response from Instructor.
+     *
+     * @return
+     * <ul>
+     * <li>true if the bundle contains response from Instructor</li>
+     * <li>false otherwise</li>
+     * </ul>
+     */
+    public boolean containsResponseFromInstructor() {
+        for (FeedbackResponseAttributes response : responses) {
+            String giverIdentifier = response.giver;
+            // Some parts of the code handles a participant as a Student if participant is both Student and Instructor
```

Comment:
If a instructor is also a student, the response should be considered as student's response.

I believe this comment is better :P